app/src/main/java/org/septa/android/app/systemstatus/GoToSystemStatusResultsOnClickListener.java

```diff
@@ -30,6 +38,18 @@ public void onClick(View v) {
             intent.putExtra(Constants.ROUTE_ID, routeId);
             intent.putExtra(Constants.TRANSIT_TYPE, transitType);
             intent.putExtra(statusType, Boolean.TRUE);
+
+            // track analytics about user origin
+            if (NextToArriveResultsActivity.class.equals(context.getClass())) {
```

Comment:
Comparing classes isn't really safe.

I would create a constructor that took a String as a parameter.  This string could then be the value that you pass to the 2nd argument of your method: logContentViewEvent.